spec/services/cclf/fee/interim_fee_adapter_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+require 'spec_helper'
```

Comment:
**[nit pick/not a blocker]** `rails_helper` already requires `spec_helper`, so this line should not be needed ;)

Reply:
👍 this needs tidying up all over the place